redis/src/aio/connection_manager.rs

```diff
@@ -123,12 +134,14 @@ impl ConnectionManager {
         exponent_base: u64,
         factor: u64,
         number_of_retries: usize,
+        max_delay: Option<u64>,
```

Comment:
instead of adding another field to the function, which is a breaking change, you should add another function taking that field.
and in order to limit the number of functions, please follow the example [in this PR](https://github.com/redis-rs/redis-rs/pull/1167) - that is,  add a configuration object, and a function using that configuration object, so that new fields won't break users.

Reply:
Thank you, i will update code with new PR